mesh/storage.c

```diff
@@ -504,7 +456,10 @@ bool storage_load_nodes(const char *dir_name)
 		if (parse_config(name_buf, filename, node_id)) {
 			remove(filename);
 			rename(name_buf, filename);
+			l_info("backup parsed successfully");
```

Comment:
to bym chyba zamienil na l_debug - moze sie czesto wykonywac a informuje o sukcesie

Reply:
Done